Examples/Tests/nuclear_fusion/analysis_two_product_fusion.py

```diff
@@ -405,7 +407,7 @@
     ds_start = yt.load(filename_start)
     ad_end = ds_end.all_data()
     ad_start = ds_start.all_data()
-    dt = float(ds_end.current_time - ds_start.current_time)
+    dt = float(ds_end.current_time - ds_start.current_time)  # noqa
```

Comment:
@dpgrote wrote in https://github.com/ECP-WarpX/WarpX/pull/5123#discussion_r1712446795

> The `eval` below needs dt. The `eval` is kind of hacky anyway. Something like this would be better
> ```py
> if i == 1:
>     specific_check1(data, dt)
> elif i == 2:
>     specific_check2(data, dt)
> ```

Reply:
@NeilZaim @RemiLehe can you quickly see how to remove the `eval` with a more explicit or handling in this function?

Reply:
A simple fix would be to do
```
eval("specific_check"+str(i))(data, dt)
```
But it would be better to remove the eval altogether.

Reply:
Yes, that looks very similar to me :sweat_smile: